static/js/main.js

```diff
@@ -1,48 +1,42 @@
-document.addEventListener("DOMContentLoaded", function(){
-  var toggle = document.getElementById("scheme-toggle");
-
-  var scheme = "light";
-  var savedScheme = localStorage.getItem("scheme");
-
-  var container = document.getElementsByTagName("html")[0];
-  var prefersDark = window.matchMedia("(prefers-color-scheme: dark)").matches;
-
-  if (prefersDark) {
-    scheme = "dark";
-  }
+function toggleDarkMode() {
+    var theme = localStorage.getItem('scheme');
+    var toggle = document.getElementById('scheme-toggle');
+    var container = document.getElementsByTagName("html")[0];
```

Comment:
_Minor/Style_: `'` and `"`should be consistent.

Reply:
I'll update this.